src/lib/jsonParseV2/parseBatchQuery.cpp

```diff
@@ -104,7 +104,7 @@ std::string parseBatchQuery(ConnectionInfo* ciP, BatchQuery* bqrP)
       std::string r = parseAttributeList(ciP, iter, &bqrP->attributeV);
 
       if (r != "OK")
-      {        
+      {
```

Comment:
Not actually related with this PR... but I spotted (this an some similar ones) during last commit, so I fixed.

NTC